app/models/spree/taxon.rb

```diff
@@ -21,19 +21,15 @@ def applicable_filters
 
     # Return meta_title if set otherwise generates from root name and/or taxon name
     def seo_title
-      if meta_title
-        meta_title
-      else
-        root? ? name : "#{root.name} - #{name}"
-      end
+      meta_title || (root? ? name : "#{root.name} - #{name}")
     end
 
     def set_permalink
-      if parent.present?
-        self.permalink = [parent.permalink, permalink_end].join('/')
-      elsif permalink.blank?
-        self.permalink = UrlGenerator.to_url(name)
-      end
+      self.permalink = if parent.present?
+                         [parent.permalink, permalink_end].join('/')
+                       elsif permalink.blank?
+                         UrlGenerator.to_url(name)
+                       end
```

Comment:
There are a few formatting changes in this commit which seem unrelated. It's better to commit these separately to show clearly what the intention of the commit is.

Reply:
I'll revert this